src/java/org/apache/cassandra/db/commitlog/CommitLogDescriptor.java

```diff
@@ -220,7 +223,20 @@ public String fileName()
 
     public String cdcIndexFileName()
     {
-        return FILENAME_PREFIX + version + SEPARATOR + id + "_cdc.idx";
+        return FILENAME_PREFIX + version + SEPARATOR + id + INDEX_FILENAME_SUFFIX;
+    }
+
+    /**
+     * Infer the corresponding cdc index file using its cdc commitlog file
+     * @param cdcFile
+     * @return cdc index file
+     */
+    public static File inferCdcIndexFile(File cdcFile)
```

Comment:
nit: consider renaming the variable `cdcCommitLogSegment` or something to disambiguate in the body of the method.